ansible/roles/software/ckan/catalog/ckan-app/defaults/main.yml

```diff
@@ -59,6 +59,9 @@ catalog_enable_clean_harvest_logs: false
 catalog_log_dir: /var/log/ckan
 catalog_gunicorn_error_log: "{{ catalog_log_dir }}/gunicorn.log"
 
+googleanalytics_ids:
+  - UA-XXXXXXX-1
```

Comment:
This should default to an empty list `[]`.

Reply:
Fixed